src/components/EditProfile.vue

```diff
@@ -122,40 +123,62 @@ export default {
         this.userProfile.email = this.UserProfile.email
       }
     },
-    update (files) {
-      // Pull image data needed for new image request
-      var imageReq = {checksum: '', extension: ''}
-      // Grab checksum and extension
-      imageReq.checksum = Crypto.MD5(files[0]).toString()
-      imageReq.extension = files[0].type.substring(6)
-      console.log(imageReq.checksum)
-      this.profileImage = files[0]
+    update (event) {
+      let vm = this
+      let file = event.target.files[0]
+      let picDisplayer = new FileReader()
+      picDisplayer.onload = (function (file) {
+        return function (event) {
+          let picFile = event.target
+          let preview = document.getElementById('preview')
+          preview.innerHTML += "<img class='thumbnail' src='" + picFile.result + "'" +
+            "title='" + file.name + "' width='150px' height='150px' style='display: inline'/>"
+        }
+      })(file)
+      vm.profileImage = file
       // upload data to the server
-      axios({
-        method: 'post',
-        url: '/api/image/new',
-        headers: {
-          'Authorization': 'Bearer ' + this.getToken
-        },
-        data: imageReq
-      })
-        .then(res => {
-          // Save image Url and ID for later image saving and profile saving
-          this.updateUser.url = res.data.url
-          this.updateUser.picid = res.data.picid
-        })
-        .catch(err => {
-          console.log(err)
-        })
+      let picHasher = new FileReader()
+      picHasher.onload = (function (file) {
+        return function (event) {
+          // let picFile = event.target
+          axios({
+            method: 'post',
+            url: '/api/image/new',
+            data: {
+              // checksum: Crypto.MD5(picFile.result).toString(),
+              extension: file.type.split('/')[1]
+            },
+            headers: {
+              'Authorization': 'Bearer ' + vm.getToken
+            }
+          })
+            .then(res => {
+              // Save image Url and ID for later image saving and profile saving
+              vm.updateUser.url = res.data.url
+              vm.updateUser.picid = res.data.picid
+            })
+            .catch(err => {
+              console.log(err)
+            })
+        }
+      })(file)
+      // Read the image
+      picDisplayer.readAsDataURL(file)
+      picHasher.readAsBinaryString(file)
     },
     saveImage: function () {
       // Save the uploaded profile picture
       axios({
         method: 'put',
         url: this.updateUser.url,
+        // headers: {
+        //  'Content-MD5': this.profileImage.md5,
+        //  'Content-Type': this.profileImage.file.type
+        // },
```

Comment:
Wasn't sure from yesterday, was going to ask @jasonmoore30 about it but if everything works on dev and prod, I'll go ahead and erase it.  I'll push up the revisions. Shouldn't be an issue.